interfaces/nginx_route/v0/schemas/requirer.py

```diff
@@ -0,0 +1,147 @@
+# generated by datamodel-codegen:
+#   filename:  requirer.json
+#   timestamp: 2023-04-12T15:19:34+00:00
+
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class NginxRouteRequirerSchema(BaseModel):
+    service_hostname: str = Field(
+        ...,
+        alias='service-hostname',
+        description='The hostname of the service to create an ingress for.',
+        examples=[],
```

Comment:
Should examples be provided or perhaps we just don't put the argument?

Reply:
I add some simple examples to the schema definition, thanks!